xml/hwbp_registers.xml

```diff
@@ -104,14 +104,34 @@
     </register>
 
     <register name="Trigger Control" short="tcontrol" address="0x7a5">
-        This optional register is one solution to a problem regarding triggers
-        with action=0 firing in M-mode trap handlers. See
-        Section~\ref{sec:mmtrigger} for more details.
+        This optional register is only writable in M mode and Debug Mode and
+        provides various control bits related to triggers.
 
-        <field name="0" bits="XLEN-1:8" access="R" reset="0" />
+        <field name="0" bits="XLEN-1:10" access="R" reset="0" />
+        <field name="hcxe" bits="9" access="WARL" reset="0">
+            Hcontext enable.
```

Comment:
"H-mode context enable." Seems just a little more clear.

Reply:
Alternatively, it's referring to the register so you could use \RcsrHcontext.

Reply:
done